recipes/tjpcov/meta.yaml

```diff
@@ -0,0 +1,109 @@
+{% set name = "tjpcov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tjpcov-{{ version }}.tar.gz
+  sha256: fd4cd437105521e11c7f977d522cdfae75fed6d7e0c06afa599cc7eec8d7a4eb
+
+build:
+  skip: true  # [win or py<38]
```

Comment:
These packages look like could be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Ok I can make this change.  The issue with Windows is the [healpy](https://healpy.readthedocs.io/en/latest/) package.  The developers have said they do not intend to support Windows.

Reply:
@ocefpaf 
This is now failing for windows with error:

> Could not solve for environment specs
The following packages are incompatible
\u251c\u2500 healpy does not exist (perhaps a typo or a missing channel);
\u2514\u2500 pyccl >=3.0.0  does not exist (perhaps a typo or a missing channel).

Reply:
The Windows failure is fine. Solver failures means this package won't be installable on Windows and we do same resources by making this noarch.

Reply:
@ocefpaf Is this good to be merged then?